introduction.md

```diff
@@ -1,2 +1,2 @@
-# I'm cool
-# I like the Ruby path the most.
+* I'm cool
+* I like the Ruby path the most.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?